File: src/test/java/no/nav/syfo/testhelper/UserConstants.java
package no.nav.syfo.testhelper;

import no.nav.syfo.mocks.AktoerMock;

public class UserConstants {

    public static final String ARBEIDSTAKER_FNR = "12345678912";
    public static final String ARBEIDSTAKER_AKTORID = AktoerMock.mockAktorId(ARBEIDSTAKER_FNR);
    public static final String LEDER_FNR = "12987654321";
    public static final String LEDER_AKTORID = AktoerMock.mockAktorId(LEDER_FNR);
    public static final String VIRKSOMHETSNUMMER = "123456789";
    public static final String NAV_ENHET = "0330";
    public static final String VEILEDER_ID = "Z999999";

    public static final String PERSON_NAME_FIRST = "First";
    public static final String PERSON_NAME_MIDDLE = "Middle";
    public static final String PERSON_NAME_LAST = "Last";

    public static final String PERSON_FULL_NAME = PERSON_NAME_FIRST + " " +  PERSON_NAME_MIDDLE + " " + PERSON_NAME_LAST;
}

File: src/test/kotlin/no/nav/syfo/testhelper/UserConstants.kt
package no.nav.syfo.testhelper

import no.nav.syfo.testhelper.mockAktorId

object UserConstants {
    const val ARBEIDSTAKER_FNR = "12345678912"
    @JvmField
    val ARBEIDSTAKER_AKTORID = mockAktorId(ARBEIDSTAKER_FNR)
    const val LEDER_FNR = "12987654321"
    @JvmField
    val LEDER_AKTORID = mockAktorId(LEDER_FNR)
    const val VIRKSOMHETSNUMMER = "123456789"
    const val NAV_ENHET = "0330"
    const val VEILEDER_ID = "Z999999"
    const val PERSON_TLF = "test@nav.no"
    const val PERSON_EMAIL = "12345678"
    const val PERSON_NAME_FIRST = "First"
    const val PERSON_NAME_MIDDLE = "Middle"
    const val PERSON_NAME_LAST = "Last"
    const val PERSON_FULL_NAME = "$PERSON_NAME_FIRST $PERSON_NAME_MIDDLE $PERSON_NAME_LAST"
}
